src/main/java/com/artipie/management/api/ApiPermission.java

```diff
@@ -59,7 +61,10 @@ public ApiPermission(final String line) {
 
     @Override
     public boolean allowed(final Authentication.User user) {
-        final Matcher matcher = PTN_PATH.matcher(new RequestLineFrom(this.line).uri().getPath());
-        return matcher.matches() && user.name().equals(matcher.group("user"));
+        final String path = new RequestLineFrom(this.line).uri().getPath();
+        final Matcher matcher = PTN_PATH.matcher(path);
+        return matcher.matches() && user.name().equals(matcher.group("user"))
+            || Stream.of(FromRqLine.RqPattern.values()).map(FromRqLine.RqPattern::pattern)
+                .anyMatch(pattern -> pattern.matcher(path).matches());
```

Comment:
@olenagerasimova I do not really understand why we need this `RqPattern` enum. Why it is not just a list of strings or `Pattern` instances? What do you think about simplifying it?

Reply:
@olegmoz I think it will be more obvious and transparent this way, all end points of artifactory API listed here. I'm planning to use these enum items in `ArtipieApi` slice in artipie as well insted of the strings.

Reply:
@olenagerasimova ok, got it